### PR TITLE
DIP-18

### PR DESCRIPTION
**Pull request by cbergz**
Changes to be made:
~~1. Snapshot vote results~~
~~2. DYDX amount~~
~~3. Ops Trust Agreement (maybe?)~~

### CODE REVIEW

File: content/dips/DIP-18.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+The snapshot vote concluded with X voters and XM DYDX in agreement (X%). The vote results have shown significant support for the program.
+https://snapshot.org/#/dydxgov.eth/proposal/0xa710b0ca85e04ac2db123ae522ac5030a008ccbc8d4d413e83d18f6032b39758
+
+**Relevant Links**
+
+DRC discussion: https://commonwealth.im/dydx/discussion/7620-drc-dydx-operations-trust
+
+## Motivation
+
+dYdX is facing an important decision to scale the protocol thoughtfully as it transitions to full community ownership. As a community, we have to look ahead towards organizational transformation through the use of subDAOs. Accomplishing this goal will first require guidance on structural implementation.
```
Comment (Jwatts15) [inline content/dips/DIP-18.md:33]: "dYdX is facing an important decision to scale the protocol thoughtfully as it transitions to full community **control**."

File: content/dips/DIP-18.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Speaking from experience with the Grants Trust, launching a subDAO is not quite as simple as opening a wallet and getting community funding. There are tons of legal procedures, operational headaches, and other nuances to building a truly internet-native organization. Community members should be able to benefit from the know-how and experience gained through the process that led to the creation and operation of the Grants Trust.
+
+To keep up with the competitive landscape, the DAO will also need to operate quickly. Doing so will require a standardized process for launching new subDAOs in charge of important functions. The playbook to be written by the Ops Trust will give the community a thoughtful approach for quickly spinning up new subDAOs, including all legal and operational procedures necessary.
+
+## Specification
+
+**The subDAO will be funded for a period of 6 months**
+
+Reverie will work with outside legal counsel to establish a Guernsey Purpose Trust. The Trust will be launched with the following contributors:
```
Comment (Jwatts15) [inline content/dips/DIP-18.md:43]: Launched --> formed

File: content/dips/DIP-18.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+This formula will be used to calculate the price: Σ(Price * Volume) / Σ(Volume).
+
+From the data, we find: $X / X = $X.
+
+The DYDX Amount to be transferred will be $360,000 / $X = **X** (rounded up to the nearest thousand for simplicity).
+
+
+## Rationale
+
+The community has addressed certain concerns and issues with a dYdX Operations subDAO run by Reverie in the DRC, but overall consensus has been in strong support of launching. The dYdX Ops subDAO will act as launching pad for the growth of a subDAO run dYdX DAO.
```
Comment (Jwatts15) [inline content/dips/DIP-18.md:83]: "The community has addressed certain concerns and issues with a dYdX Operations subDAO run by Reverie in the DRC, but overall consensus has been in strong support of launching."

A little unclear could you please revise.

**Review by Jwatts15 — APPROVED** [review]

**Review by Jwatts15 — APPROVED** [review]

**Review by dgogel — APPROVED** [review]
Reviewed DIP

**Pull request merged**